src/main/resources/db/migration/common/V1_287__failure_empty_file.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO object_record_status (ors_id, ors_description) VALUES (20, 'Empty File Failed');
```

Comment:
FYI I think there's an expectation from @barry-jones50410 / DBA team for us to maintain a record of standing data records in `resources/db/reference/modernised_darts_standing_data.sql`?